src/main/kotlin/no/nav/syfo/service/VarselSenderService.kt

```diff
@@ -0,0 +1,62 @@
+package no.nav.syfo.service
+
+import no.nav.syfo.db.DatabaseInterface
+import no.nav.syfo.db.domain.PPlanlagtVarsel
+import no.nav.syfo.db.domain.VarselType
+import no.nav.syfo.db.fetchMerVeiledningVarslerToSend
+import no.nav.syfo.db.fetchPlanlagtVarselByUtsendingsdato
+import no.nav.syfo.db.fetchUtsendteMerVeiledningVarslerSiste3Maneder
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+class VarselSenderService(
+    private val databaseAccess: DatabaseInterface,
+    private val sykmeldingService: SykmeldingService
+) {
+    private val log = LoggerFactory.getLogger("no.nav.syfo.service.VarselSenderService")
+
+    suspend fun getVarslerToSendToday(): List<PPlanlagtVarsel> {
```

Comment:
Er vi nødt til å ha `suspend` på denne? Blir så knotete å teste..

Reply:
Ja, pga sjekk for aktiv sykmelding.